ux-guidelines.md

```diff
@@ -0,0 +1,178 @@
+date (e.g., "1234 01/32).
+
+Note: An implementor may truncate the `displayName` and `details` fields in
+order to fit the text into the available UX space. See issue #269 for
+discussions on setting normative length limits on fields including payeeName
+and payeeOrigin.
+
+The `icon` field is set to the URL of an image representing the payment
+instrument. Implementors are expected to support common image formats such as
+PNG or JPG. The payment instrument icon image is intended to be a quick visual
+confirmation for the user, and is expected to be displayed fairly small.
+
+Implementation specifics:
+
+- Google Chrome's (and other Chromium-based browsers) implementation on both
+  desktop and mobile platforms currently linearly scales the image (preserving
+  the input aspect ratio) to fit in a 32dp x 20dp region, e.g. the display
+  region has an aspect ratio of 1.6:1.
+    - Given current average screen densities, we would recommend that
+      integrators pass in a logo with physical pixel size at least 3x of that,
+      e.g. minimal 96px by 60px.
+    - The Chrome team have stated that they are interested in hearing from
+      partners if a 32dp x 20dp region is too small, as it may not allow for
+      detailed icons (such as full card art for credit cards).
+
+### Payment entity logos
+
+Relevant fields: [paymentEntitiesLogos], [paymentEntitiesLogos\[x\].url], [paymentEntitiesLogos\[x\].label]a
+
+The payment entity logos are intended to communicate to the user the identity
+of payment companies or organizations that are involved in the current
+transaction, who might otherwise not be clear to the user. Examples include the
+issuing bank and/or a card network in the case of credit/debit card payments,
+the user's bank and a payment initiation services provider (PISP) in Open
+Banking, etc.
+
+The payment entity logos are expected to be displayed prominently within the
+SPC transaction confirmation UX, but should not significantly overweight the
+other information shown. Implementors are able to support any number of logos
+passed within `paymentEntityLogos`, and may ignore 'extra' logos beyond what
+they support displaying, but it is recommended that they support up to two
+logos being shown.
+
+Each payment entity logo has two fields.
+
+The `label` field is used to convey a textual description of the payment entity
+to the user. This field may be displayed by the implementor, or it may be used
+only for accessibility purposes (e.g., for a screen reader to describe the
+logo). It is recommended that this text be short but descriptive to the user,
+e.g. "FancyBank logo".
+
+The `url` field is set to the URL of a payment entity logo image. Implementors
+are expected to support common image formats such as PNG or JPG. Logos should
+generally be "wordmark" or combination logos, which often display more clearly
+to users than 'simple' square logos. Logos should also have transparent
```

Comment:
You use both double quotes and single quotes in the sentence. Since you've been using single quotes elsewhere, suggest 'wordmark.'

Reply:
It looks like I mostly use double quotes elsewhere? In any case, I just removed the word 'simple' instead.